_locales/ar/messages.json

```diff
@@ -550,11 +550,11 @@
     "description": "Pref tab text"
   },
   "paused": {
-    "message": "ملبث",
+    "message": "متوقف",
     "description": "Status text; manager"
   },
   "pause_download": {
-    "message": "ألبِث",
+    "message": "إيقاف التنزيل",
```

Comment:
Same as above. Plus not respecting the verb type used across the addon translation.